packages/reporters/bundle-analyzer/client/index.js

```diff
@@ -100,3 +100,12 @@ function debounce(fn, delay) {
 function translate3d(x, y, z) {
   return `translate3d(${x}px, ${y}px, ${z}px)`;
 }
+
+function formatSize(x){
+  const units = ['bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+  let l = 0, n = parseInt(x, 10) || 0;
+  while(n >= 1024 && ++l){
+      n /= 1024;
+  }
+  return(`${n.toFixed(n < 10 && l > 0 ? 1 : 0)} ${units[l]}` + (l ? ` (${x} bytes)` : ''));
+}
```

Comment:
Parcel is using [`filesize`](https://github.com/parcel-bundler/parcel/blob/18b038d57043965afb31077db1f46dadb11a6a78/packages/reporters/cli/package.json#L39), therefore it would be better to have the same behavior:

```suggestion
function formatSize(x){
  const units = ['B', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
  let l = 0, n = parseInt(x, 10) || 0;
  while(n >= 1000 && ++l){
      n /= 1000;
  }
  return(`${n.toFixed(n < 10 && l > 0 ? 1 : 0)} ${units[l]}`);
}
```